frontend/src/lib/components/SelectGradientOverflow.tsx

```diff
@@ -20,6 +21,27 @@ function DropdownGradientRenderer({
     return <div ref={innerRef}>{menu}</div>
 }
 
+/**
+ * Ant Design does not export the component used in multi-select lists,
+ * so we're recreating it with a title property for better UX
+ */
+type CustomTagProps = Parameters<Exclude<SelectProps<any>['tagRender'], undefined>>[0]
+
+function CustomTag({ label, onClose, value }: CustomTagProps): JSX.Element {
+    return (
+        <div className="ant-select-selection-overflow-item">
+            <span className="ant-select-selection-item">
+                <span className="ant-select-selection-item-content" title={value.toString()}>
```

Comment:
If we're going to do this entire override, might be worth using `Tooltip` component instead of browser's title? It'd be a matter of testing to see if this provides better UX, but just a hunch.

Also, if I understand correctly the reason to re-write this code is that the component cannot be imported? Becuase a simple wrap in a `Tooltip` will definitely make for more cleaner/reusable code.

Reply:
Rewrote this to use a `Tooltip` wrapped around a `Tag` with custom CSS